docker/policies/run_using_apt_upgrade.rego

```diff
@@ -0,0 +1,38 @@
+package appshield.DS006
+
+__rego_metadata__ := {
+	"id": "DS006",
+	"title": "Run Using Upgrade Commands",
+	"version": "v1.0.0",
+	"severity": "CRITICAL",
+	"type": "Dockerfile Security Check",
+	"description": "Commands 'apt-get upgrade' and 'apt-get dist-upgrade' should not be used",
+	"recommended_actions": "You should not even use 'apt-get upgrade' and 'apt-get dist-upgrade'.",
+	"url": "https://docs.docker.com/develop/develop-images/dockerfile_best-practices/",
```

Comment:
I didn't see this rule in the best practice. Where does it say "upgrade" should not be used?

Reply:
original:
https://github.com/Checkmarx/kics/blob/master/assets/queries/dockerfile/run_using_upgrade_commands/metadata.json

Reply:
also found
https://pythonspeed.com/articles/security-updates-in-docker/

Reply:
Thanks! As the article says, we should not have this policy.